src/models/game.ts

```diff
@@ -111,7 +111,7 @@ class Game extends Model<InferAttributes<Game>, InferCreationAttributes<Game>> {
 
 Game.init({
   id: {
-    type: DataTypes.INTEGER.UNSIGNED,
+    type: DataTypes.INTEGER,
```

Comment:
This silences a warning we've been getting which says postgres does not support unsigned int types.